apps/OboeTester/app/src/main/java/com/mobileer/oboetester/OboeAudioStream.java

```diff
@@ -245,6 +246,11 @@ public int getDeviceId() {
     }
     private native int getDeviceId(int streamIndex);
 
+    public int[] getDeviceIds() {
+        return getDeviceIds(streamIndex);
+    }
+    private native int[] getDeviceIds(int streamIndex);
```

Comment:
```@Nullable``` annotation might make this safer.
Or always return an array, that might be empty.

Reply:
Thanks! I added @Nullable